IPython/html/static/base/less/page.less

```diff
@@ -127,3 +132,9 @@ span#login_widget > .button,
     .fa();
     content: @ico;
 }
+
+@media (min-width: @screen-sm-min){
+    .container {
```

Comment:
Because otherwise I need to maintain a patch in `componenents/bootstrap/`.

I'm fine with dropping this if you don't like, but it's the part that make the notebook not jump from 768px width to 720 px width when you grow the window width from 768px to 769px.